<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServicesWidget</class>
 <widget class="QWidget" name="ServicesWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="qss_tags" stdset="0">
   <stringlist notr="true">
    <string>xlet_frame</string>
   </stringlist>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="services_label">
     <property name="text">
      <string>Services</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="qss_tags" stdset="0">
      <stringlist notr="true">
       <string>xlet_h1</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="call_filtering_checkbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Call &amp;filtering</string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dnd_checkbox">
       <property name="text">
        <string>Do not &amp;disturb</string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="call_forwards_label">
     <property name="text">
      <string>Call forwards</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="qss_tags" stdset="0">
      <stringlist notr="true">
       <string>xlet_h2</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="fwdsimple_radiobutton">
       <property name="text">
        <string>Simple call forwards</string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QFrame" name="fwdunc_input_button_group">
       <property name="styleSheet">
        <string notr="true">border: </string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>input_button</string>
        </stringlist>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="fwdunc_input">
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fwdunc_button">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../xivoclient.qrc">
            <normaloff>:/images/search-regular.svg</normaloff>:/images/search-regular.svg</iconset>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>black</string>
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="fwdunc_radiobutton">
       <property name="text">
        <string>&amp;Unconditional forward to</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="nofwd_radiobutton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>No call forward</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="3">
      <widget class="QFrame" name="fwdna_input_button_group">
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>input_button</string>
        </stringlist>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="fwdna_input">
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fwdna_button">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../xivoclient.qrc">
            <normaloff>:/images/search-regular.svg</normaloff>:/images/search-regular.svg</iconset>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>black</string>
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="3">
      <widget class="QFrame" name="fwdbusy_input_button_group">
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>input_button</string>
        </stringlist>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="fwdbusy_input">
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fwdbusy_button">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../xivoclient.qrc">
            <normaloff>:/images/search-regular.svg</normaloff>:/images/search-regular.svg</iconset>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>black</string>
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="fwdbusy_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Forward on &amp;busy to</string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_sub_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="fwdna_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Forward on &amp;no answer to</string>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_sub_list</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>call_filtering_checkbox</tabstop>
  <tabstop>dnd_checkbox</tabstop>
  <tabstop>nofwd_radiobutton</tabstop>
  <tabstop>fwdunc_radiobutton</tabstop>
  <tabstop>fwdunc_input</tabstop>
  <tabstop>fwdunc_button</tabstop>
  <tabstop>fwdsimple_radiobutton</tabstop>
  <tabstop>fwdna_checkbox</tabstop>
  <tabstop>fwdna_input</tabstop>
  <tabstop>fwdna_button</tabstop>
  <tabstop>fwdbusy_checkbox</tabstop>
  <tabstop>fwdbusy_input</tabstop>
  <tabstop>fwdbusy_button</tabstop>
 </tabstops>
 <resources>
  <include location="../../../xivoclient.qrc"/>
 </resources>
 <connections/>
</ui>
